src/Sylius/Bundle/AttributeBundle/Form/Type/AttributeType/Configuration/ChoiceAttributeConfigurationType.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\AttributeBundle\Form\Type\AttributeType\Configuration;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+/**
+ * @author Saša Stamenković <umpirsky@gmail.com>
+ */
+class ChoiceAttributeConfigurationType extends AbstractType
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('options', 'collection', [
```

Comment:
How should we render this? Should we have separate template for each configuration type?